### PR TITLE
Add materials to the workOrders.draft endpoint

### PR DESCRIPTION
**Pull request by WouterSioen**

### CODE REVIEW

**Review by tiagobastos — APPROVED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by JorenSaey — DISMISSED** [review]

**Review by mikeverf — DISMISSED** [review]

**Review by kevinwaelkens — DISMISSED** [review]

**Review by dietrichm — DISMISSED** [review]

**Comment by dietrichm** [comment]
✅ Checked

File: apiary.apib [hunk 1]
```diff
@@ -3302,6 +3302,10 @@ Add a draft work order.
        + description: `Pipework intervention` (string, optional)
        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
+       + products (array, optional)
+           + (object)
+               + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
+               + quantity: `3` (number, required)
```
Comment (JordyDeGroeve) [inline apiary.apib]: Q: shouldn't an integer be documented without quotes?

**Review by tiagobastos — DISMISSED** [review]

**Review by JordyDeGroeve — DISMISSED** [review]

**Review by mikeverf — DISMISSED** [review]

**Review by dietrichm — DISMISSED** [review]

**Comment by dietrichm** [comment]
✅ Checked

**Review by kevinwaelkens — DISMISSED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -3302,6 +3302,10 @@ Add a draft work order.
        + description: `Pipework intervention` (string, optional)
        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
+       + products (array, optional)
```
Comment (jenssegers) [inline apiary.apib]: Would like to give `material` a second thought, but only if it fits of course.

**Review by jenssegers — APPROVED** [review]

File: apiary.apib (hunk 2 above)
Comment (jenssegers) [inline apiary.apib]:
```suggestion
       + material (array, optional)
```

File: src/01-general/work-orders.apib [hunk 3]
```diff
@@ -17,6 +17,10 @@ Add a draft work order.
        + description: `Pipework intervention` (string, optional)
        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
+       + products (array, optional)
```
Comment (jenssegers) [inline src/01-general/work-orders.apib]:
```suggestion
       + material (array, optional)
```

**Review by jenssegers — DISMISSED** [review]

**Comment by WouterSioen** [comment]
@jenssegers I double checked this with @dietrichm who double checked it with @charlottevb and we're going to stay with products for now.

**Review by JordyDeGroeve — APPROVED** [review]

**Review by jenssegers — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by mikeverf — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Pull request merged**